app/main/presenters/search_results.py

```diff
@@ -27,3 +27,13 @@ def _add_highlighting(self, service):
                 service['serviceSummary'] = Markup(
                     ''.join(service['highlight']['serviceSummary'])
                 )
+
+
+class AggregationResults(object):
+    """Provides access to the aggregation results information"""
+
+    def __init__(self, response):
+        self.results = response['aggregations']
+        self.total = response['meta']['total']
+        if 'page' in response['meta']['query']:
+            self.page = response['meta']['query']['page']
```

Comment:
On second inspection... There are tests asserting that page isn't set, so I will leave it as-is for now because it's 'expected' behaviour.